### DISCUSSION TITLE
监控模版共享

### DISCUSSION THREAD

**Discussion by zhangwenjingqwe** [apache/hertzbeat#1986] [Ideas]
大佬有没有想过将监控模版搞个社区进行共享，大家有什么好的监控模版也可以在社区上传，一起壮大系统的模版库，现在的监控种类还是不太多，对一些传统的设备支持不太够，像是一些网络安全的设备都没有监控模版，需要用户自己去维护。

**Comment by zuobiao-zhou** [comment]
This is [a task of OSPP](https://summer-ospp.ac.cn/org/prodetail/24bef0159?lang=zh&list=pro). If you are interested, welcome to participate.
Here is the linked issue. https://github.com/apache/hertzbeat/issues/1792
